Add "Prototyping" working group

morph-dev opened the pull request:
"Protocol Prototyping" is a new team formed during EF reorg.

We work on the EIPs that are in the late research (ideally) and/or early implementation phase. We help with initial prototyping, spec clarification and adjustment.

In this PR, I created new working group (under _Wayfinding_ category) and moved team members from other working groups. No members were added or removed.

jochem-brouwer commented on docs/01-membership.md:
```diff
@@ -36,30 +36,32 @@ Individuals from active working groups produce the membership by opting into Pro
 | [Thomas Thiery](https://github.com/soispoke/) | 1 | [ethresear.ch/u/soispoke/summary](https://ethresear.ch/u/soispoke/summary/) |
+| **Prototyping** (4 contributors) | | |
+| [Danno Ferrin](https://github.com/shemnon/) | 1 | [ethereum/EIPs](https://github.com/ethereum/EIPs/pulls?q=author%3Ashemnon), [ethereum/execution-spec-tests](https://github.com/ethereum/execution-spec-tests/pulls?q=author%3Ashemnon), [hyperledger/besu](https://github.com/hyperledger/besu/pulls?q=author%3Ashemnon) |
+| [Jochem](https://github.com/jochem-brouwer/) | 1 | [ethereumjs/ethereumjs-monorepo](https://github.com/ethereumjs/ethereumjs-monorepo/pulls?q=is%3Apr+author%3Ajochem-brouwer+) |
```
```suggestion
| [Jochem](https://github.com/jochem-brouwer/) | 1 | [ethereum/EIPs](https://github.com/ethereum/EIPs/pulls?q=author%3Ajochem-brouwer), [ethereum/execution-spec-tests](https://github.com/ethereum/execution-spec-tests/pulls?q=author%3Ajochem-brouwer), [ethereumjs/ethereumjs-monorepo](https://github.com/ethereumjs/ethereumjs-monorepo/pulls?q=is%3Apr+author%3Ajochem-brouwer+) |
```

shemnon commented on docs/01-membership.md:
```diff
@@ -36,30 +36,32 @@ Individuals from active working groups produce the membership by opting into Pro
 | [Mike Neuder](https://github.com/michaelneuder) | 1 | |
 | [Thomas Thiery](https://github.com/soispoke/) | 1 | [ethresear.ch/u/soispoke/summary](https://ethresear.ch/u/soispoke/summary/) |
+| **Prototyping** (4 contributors) | | |
+| [Danno Ferrin](https://github.com/shemnon/) | 1 | [ethereum/EIPs](https://github.com/ethereum/EIPs/pulls?q=author%3Ashemnon), [ethereum/execution-spec-tests](https://github.com/ethereum/execution-spec-tests/pulls?q=author%3Ashemnon), [hyperledger/besu](https://github.com/hyperledger/besu/pulls?q=author%3Ashemnon) |
```
While my code will show up in a variety of repos, I don't consider myself as having a commitment to any specific project right now 
```suggestion
| [Danno Ferrin](https://github.com/shemnon/) | 1 | |
```

jochem-brouwer reviewed (approved):
Thanks :smile: :+1: 

(will also explicitly approve here)

morph-dev commented:
Rebased the changes on top of the head.
I also added other members of the team (that joined in the meantime), and updated/fixed number of contributors for each working group.

@CarlBeek @shemnon @jochem-brouwer @nerolation @raxhvl Can you check the changes and give :+1: on this comment if there are no concerns.

raxhvl reviewed (approved):
Thanks @morph-dev!

morph-dev commented:
@cheeky-gorilla @tvanepps 
Everybody affected either approved or gave a :+1: . Can we get this merged?
